app/code/Magento/Directory/Block/Data.php

```diff
@@ -168,11 +168,12 @@ public function getRegionCollection()
         }
         return $collection;
     }
-
+    
     /**
-     * Returns region html select
-     *
      * @return string
+     * 
+     * @deprecated
+     * @see getRegionSelect() method for more configuration
```

Comment:
Please, `* @see getRegionSelect() method for more configuration` -> `     * @see getRegionSelect() method for more configurations `